src/fusor/examples/bcr_abl1.json

```diff
@@ -1,111 +1,131 @@
-              "value": 23253981
+      "end": 23290413,
+      "extensions": [
+        {
+          "name": "NM_004327.4:e._14",
+          "description": "VICC exon representation of the aligned transcript boundary.",
+          "value": {
+            "exon_end": 14,
+            "exon_end_offset": 0,
+            "sequenceReference":{
+              "type": "SequenceReference",
+              "id": "NM_004327.4",
+              "refgetAccession": "SQ.kpytJsXw3BwLC3oBSjHQS1kwxs4WO3I3",
+              "residueAlphabet": "na"
             }
           }
-        }
-      },
-      "exon_end": 2,
-      "exon_end_offset": 182
-    },
-    {
-      "type": "LinkerSequenceElement",
-      "linker_sequence": {
-        "id": "sequence:ACTAAAGCG",
-        "type": "SequenceDescriptor",
-        "sequence": "ACTAAAGCG",
-        "residue_type": "SO:0000348"
-      }
-    },
-    {
-      "type": "TranscriptSegmentElement",
-      "transcript": "refseq:NM_005157.5",
-      "exon_start": 2,
-      "exon_start_offset": -173,
-      "gene_descriptor": {
-        "id": "normalize.gene:ABL1",
-        "type": "GeneDescriptor",
-        "label": "ABL1",
-        "gene_id": "hgnc:76"
-      },
-      "element_genomic_start": {
-        "id": "fusor.location_descriptor:NC_000009.12",
-        "type": "LocationDescriptor",
-        "label": "NC_000009.12",
-        "location": {
-          "type": "SequenceLocation",
-          "sequence_id": "refseq:NC_000009.12",
-          "interval": {
-            "type": "SequenceInterval",
-            "start": {
-              "type": "Number",
-              "value": 130854064
+        },
+        {
+          "name": "NM_004327.4:c._2782",
+          "description": "Transcript SequenceLocation of the aligned transcript boundary.",
+          "value": {
+            "type": "SequenceLocation",
+            "sequenceReference": {
+              "id": "NM_004327.4",
+              "type": "SequenceReference",
+              "refgetAccession": "SQ.kpytJsXw3BwLC3oBSjHQS1kwxs4WO3I3",
+              "residueAlphabet": "na"
             },
-            "end": {
-              "type": "Number",
-              "value": 130854065
-            }
+            "end": 3234
+          }
+        },
+        {
+          "name": "gene",
+          "description": "The gene concept (BCR) associated with this fusion partner.",
+          "value": {
+            "code": "hgnc:1014",
+            "system": "https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/",
+            "label": "BCR"
           }
         }
-      }
+      ]},
+      {
+        "type": "SequenceLocation",
+        "sequenceReference": {
+          "id": "GRCh38:chr9",
+          "type": "SequenceReference",
+          "refgetAccession": "SQ.KEO-4XBcm1cxeo_DIQ8_ofqGUkp4iZhI",
+          "residueAlphabet": "na"
+        },
+        "start": 130854064,
+        "extensions": [
+          {
+            "name": "NM_005157.6:e.2_",
```

Comment:
Just wanted to clarify how to "read" this. The schema description of `name` says it "[s]hould be indicative of its meaning and/or the type of information it value represents", but instead we're using the nomenclature, ie a shorthand summary of the `value`. Maybe I'm just trying to use VRS1 glasses to look at a VRS2 object, but this seems a little off.